New log file opened at Tue Nov 17 12:05:21 ECT 2015 by 7981@toshiba
Taking next connection to: 39272
New client attempting to connect on port: 40044-39272
  Address = /127.0.1.1:39272
This client on port 40044 has uuid 2d96abb3-9d8a-43bf-93cf-78a2dffe81ec configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-2d96abb3-9d8a-43bf-93cf-78a2dffe81ec
Taking next connection to: 39272
New client attempting to connect on port: 40126-39272
  Address = /127.0.1.1:39272
Handshake exception on socket: 40126
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 39272
New client attempting to connect on port: 40127-39272
  Address = /127.0.1.1:39272
This client on port 40127 has uuid 5a081924-a33b-4342-8b6d-1719253d0a5a configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-2d96abb3-9d8a-43bf-93cf-78a2dffe81ec, play-fork-run-5a081924-a33b-4342-8b6d-1719253d0a5a
Taking next connection to: 39272
New client attempting to connect on port: 40179-39272
  Address = /127.0.1.1:39272
Handshake exception on socket: 40179
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 39272
New client attempting to connect on port: 40180-39272
  Address = /127.0.1.1:39272
This client on port 40180 has uuid fa85619f-08c3-4a0f-9b2b-7031689ef7c0 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-2d96abb3-9d8a-43bf-93cf-78a2dffe81ec, play-fork-run-fa85619f-08c3-4a0f-9b2b-7031689ef7c0
Taking next connection to: 39272
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 39272
